src/gt4py/next/program_processors/codegens/gtfn/itir_to_gtfn_ir.py

```diff
@@ -67,6 +67,27 @@ def pytype_to_cpptype(t: ts.ScalarType | str) -> Optional[str]:
 _horizontal_dimension = "gtfn::unstructured::dim::horizontal"
 
 
+def _is_tuple_of_ref_or_literal(expr: itir.Expr) -> bool:
```

Comment:
remove code duplication with the one in gtfn_ir.py

Reply:
The two functions are operating on different IRs. I could add the IR modules, e.g. `itir` or `gtfn_ir`, as a parameter, but that doesn't feel right. I would rather keep it as is and eventually get rid of the entire thing altogether.